build_all/base-libraries/AzureIoTHub/README.md

```diff
@@ -104,19 +109,24 @@ You should have the following ready before beginning with any board:
 
 4. Update IoT Hub Connection string in `iot_configs.h`
 
-5. Navigate to where your esp32 board package is located, typically in `C:\Users\<your username>\AppData\Local\Arduino15\packages` on Windows and `~/.arduino15/packages/` on Linux
+5. Configure board library using the automation script and `python3`. If you choose this method you can skip step 6.
+    - Clone this repo `git clone https://github.com/Azure/azure-iot-arduino.git` or download the script [directly](https://raw.githubusercontent.com/Azure/azure-iot-arduino/master/scripts/automate_board_config.py).
```

Comment:
Please change repo specified to the pal repo, if a user is going to make the library themself, I prefer they have access to all the tools.

Reply:
Agreed. I've updated the documentation to be a bit more user friendly and updated the repository links so it'll be available downstream and present with all the other tools in c3337f764bf7e81b7af69ff921cbfb0d8dfe742b